vizro-core/src/vizro/models/_controls/filter.py

```diff
@@ -35,6 +47,21 @@ def _filter_isin(series: pd.Series, value: MultiValueType) -> pd.Series:
     return series.isin(value)
 
 
+def _filter_date_between(series: pd.Series, value: List[str]) -> pd.Series:
+    series = pd.to_datetime(series)
+    start_date, end_date = value
+
+    return series.between(start_date, end_date, inclusive="both")
```

Comment:
I'm getting the following error here when I clear UI value from the vm.DateRangePicker:
`... not enough values to unpack (expected 2, got 1)`